### ISSUE TITLE
Subway doesn't open until 12pm on Sunday

### ISSUE THREAD

**Issue by john3parker** [hungernotimpossible/hni#153]
The Subway at this address doesn't open until 12pm on Sunday. We probably need to think about daily hours of operation. 

Subway
616 North Garland Avenue
Fayetteville, AR

This is the message received on Subway.com site when ordering at 11:48am CST on Sunday December 18.

Store Cannot Accept Remote Orders at this Time
This store is currently not accepting remote orders. You are free to browse the menu and fill your cart but you will not be able to checkout.

**Comment by KKAY99** [comment]
Yes we support the daily hours.   The website says their hours start at
10:00 a.m. on Sunday.  I wonder if they had changed their hours seasonally
or just for the bad weather?

**Comment by waynemz** [comment]
Is there a way to find out changes to the hours of operation for subway locations on a per location basis in an automated fashion? If not, the volunteers placing an order might be able to reject the order with some sort of feedback. I have not tried using the UI myself so I don't know how that works.

**Comment by KKAY99** [comment]
The stores have been updated. Second issue of cancelling order is listed as GitHub order.
